generator/src/test/java/com/scottlogic/deg/generator/reducer/ConstraintReducerTest.java

```diff
@@ -1,1309 +0,0 @@
-/////*
```

Comment:
should this be removed as part of this pr

Reply:
Was there a reason it was not removed previously?

It came up in this PR because Charlton had made changes to the commented-out code, which drew our attention to it.

Reply:
i think the tests themselves are still valid, but the assertions are way outdated, and will need to be changed completely.

this was meant to happen in an older PR, but it got missed.

Reply:
The logic this is testing is now already tested which is why I removed it